client/me/profile-gravatar/index.jsx

```diff
@@ -23,32 +26,59 @@ export default React.createClass( {
 	},
 
 	render() {
-		const profileURL = `//gravatar.com/${ this.props.user.username }`;
+		const profileURL = `https://gravatar.com/${ this.props.user.username }`;
+		// use imgSize = 400 for caching
+		// it's the popular value for large Gravatars in Calypso
+		const GRAVATAR_IMG_SIZE = 400;
+
+		if ( isEnabled( 'me/edit-gravatar' ) ) {
+			return (
+				<div className="profile-gravatar">
+					<Gravatar user={ this.props.user } size={ 150 } imgSize={ GRAVATAR_IMG_SIZE } />
+					<h2 className="profile-gravatar__user-display-name">{ this.props.user.display_name }</h2>
+					<div className="profile-gravatar__user-secondary-info">
+						<ExternalLink
+							href={ profileURL }
+							target="_blank"
+							rel="noopener noreferrer" >
+							@{ this.props.user.username }
+						</ExternalLink>
+					</div>
+				</div>
+			);
+		}
 
 		return (
 			<div className="profile-gravatar">
 				<Animate type="appear">
-					<a
+					<ExternalLink
 						href="https://secure.gravatar.com/site/wpcom?wpcc-no-close"
 						target="_blank"
 						rel="noopener noreferrer"
 						className="profile-gravatar__edit"
 						onClick={ this.recordClickEvent( 'Gravatar Update Profile Photo in Sidebar' ) } >
 
-						<Gravatar user={ this.props.user } size={ 150 } imgSize={ 400 } />
+						<Gravatar user={ this.props.user } size={ 150 } imgSize={ GRAVATAR_IMG_SIZE } />
 
 						<span className="profile-gravatar__edit-label-wrap">
 							<span className="profile-gravatar__edit-label">
-								{ this.translate( 'Update Profile Photo' ) }
+								{ this.props.translate( 'Update Profile Photo' ) }
 							</span>
 						</span>
-					</a>
+					</ExternalLink>
 				</Animate>
 				<h2 className="profile-gravatar__user-display-name">{ this.props.user.display_name }</h2>
 				<div className="profile-gravatar__user-secondary-info">
-					<a href={ profileURL } target="_blank" rel="noopener noreferrer">@{ this.props.user.username }</a>
+					<ExternalLink
+						href={ profileURL }
+						target="_blank"
+						rel="noopener noreferrer" >
+						@{ this.props.user.username }
```

Comment:
This is unlikely to happen, but getCurrentUser can return null, so it's safer to do a check here:
```jsx
user && user.username
```